### DISCUSSION TITLE
朋友无法进入到我的MC服务器

### DISCUSSION THREAD

**Discussion by GuYuLuLe** [natfrp/wiki#253] [使用问题]
我是创建好了服务器，但是朋友那边进服务器的时候是显示无法进入，说是让他重新启动游戏或者启动器，请问一下是什么原因呢，是因为我们俩个都不是正版的原因吗

**Comment by H0I00** [comment]
首先，确保两个人的游戏以及forge（fabric）版本相同，建议服主将自己的versions文件夹拷贝给自己的朋友。
其次，服主请尝试添加此mod。
https://www.mcmod.cn/class/2754.html
关闭正版验证，打开端口映射。
我习惯将获取IP关闭，这点请你自行尝试

**Comment by cohhi** [comment]
试试关闭在服务器配置文件上关闭正版验证
将onlinemode 改成 false
